addons/sourcemod/scripting/ttt/ttt_shop.sp

```diff
@@ -622,6 +641,82 @@ public Action Command_ReopenShop(int client, int args)
     return Plugin_Continue;
 }
 
+public Action Command_Inventory(int client, int args)
+{
+	if (!TTT_IsClientValid(client))
+	{
+		return Plugin_Handled;
+	}
+	
+	if (!TTT_IsRoundActive())
+	{
+		ReplyToCommand(client, "%T", "RoundInactive");
+		return Plugin_Handled;
+	}
+	
+	Menu inventoryMenu = CreateMenu(Menu_InventoryHandler);
+	StringMap playerInv = playerInventory[client][hInvItems];
+	if (playerInv == INVALID_HANDLE)
+	{
+		LogToFile(g_sLog, "Function: Command_Inventory - Inventory not intialised for client: %d.", client);
+		return Plugin_Handled;
+	}
+	
+	//any invItem[InventoryItem];
+	
+	StringMapSnapshot inventoryListing = playerInv.Snapshot();
+	for (int i = 0; i < inventoryListing.Length; i++)
+	{
+		char itemShort[16];
+		inventoryListing.GetKey(i, itemShort, sizeof(itemShort));
+		
+		int amount = -1;
+		GetTrieValue(inventoryListing, itemShort, amount);
+		if (amount <= 0)
+		{
+			continue;
+		}
+		
+		char itemLong[32];
+		if (GetItemLong(itemShort, itemLong, sizeof(itemLong)))
+		{
+			inventoryMenu.AddItem(itemShort, itemLong);
```

Comment:
Maybe add an amount indicator if the amount is greater than 1. Like "C4 - Bomb (x2)" or smth similar.